versions.properties

```diff
@@ -79,6 +79,8 @@ org.jvnet.jaxb2_commons:jaxb2-basics-ant=$jaxbBasicsVersion
 org.jvnet.jaxb2_commons:jaxb2-basics-runtime=$jaxbBasicsVersion
 org.jvnet.jaxb2_commons:jaxb2-default-value=1.1
 org.jvnet.jaxb2_commons:jaxb2-fluent-api=3.0
+org.jvnet.jaxb2_commons:jaxb2-basics=0.6.0
+org.jvnet.jaxb2_commons:jaxb2-basics-runtime=0.6.0
```

Comment:
this is already present 4 lines up. also the basics should probably use the predefine version so that they stay hooked together. and finally these are alphabetized, so basics should be above the others.